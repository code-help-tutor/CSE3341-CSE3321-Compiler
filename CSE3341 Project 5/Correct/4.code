procedure t4 and
	procedure A(x) is
		integer y;
	begin
		out(x);
	end
is
	integer a;
	integer b;
	record z;
begin
	a := 5;
	b := 10;
	z := new record[1];
	z[0] := a+b;
	begin A(z);
end
