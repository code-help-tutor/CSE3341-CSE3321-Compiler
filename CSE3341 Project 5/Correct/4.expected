gc:1
15
gc:0
